c/.Dockerfile.preprocessing-consistency

```diff
@@ -23,6 +23,11 @@ FROM ubuntu:trusty
 RUN apt-get update && apt-get install -y \
   libc6-dev-i386
 
+RUN apt-get install -y jq\
```

Comment:
This is [problematic](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get), `apt-get update` and `apt-get upgrade` need to be executed in the same `RUN` statement.

Reply:
But `apt-get upgrade` wasn't used before, either.
Regarding `apt-get update`: I know, that's why I posted above comment asking about it.

Reply:
Sorry, I meant `update` and `install`.

Regardless of any problems of upgrading `libc-dev-i386`, this rule for Dockerfiles should still be followed.

Reply:
Alright, I'll wait whether @tautschnig or someone else has any idea why the checks are failing and update it accordingly once we know what the reason is.